code/_onclick/hud/screen_objects.dm

```diff
@@ -60,6 +60,9 @@
 	var/slot_id	//The indentifier for the slot. It has nothing to do with ID cards.
 	var/hand_index
 
+/obj/screen/holomap
+	plane = HUD_PLANE
```

Comment:
They are. I originally set it to HUD_PLANE-1 but I changed it back to that in a revision a few minutes ago. I guess i can just remove that line altogether.